pkg/apis/tensorflow/v1alpha2/types.go

```diff
@@ -78,6 +83,13 @@ type TFReplicaSpec struct {
 // CleanPodPolicy describes how to deal with pods when the TFJob is finished.
 type CleanPodPolicy string
 
+// TTLAfterFinished uses a duration string to describe the timeout of
+// cleanup tf-jobs.  A duration string is a possibly signed sequence of
+// decimal numbers, each with optional fraction and a unit suffix,
+// such as "300ms", "-1.5h" or "2h45m".
+// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+type TTLAfterFinished string
```

Comment:
Lets be consistent with what K8s is doing
https://docs.google.com/document/d/1U6h1DrRJNuQlL2_FYY_FdkQhgtTRn1kEylEOHRoESTc/edit?ts=5b58ac5b#heading=h.gntasb1fv9ky

The field is called TTLSecondsAfterFinishing
If you look at the discussion in the comments they explicitly considered making it a string or using a duration to allow different units and the idea was rejected.